### ISSUE TITLE
deb-s3 delete fails with TypeError

### ISSUE THREAD

**Issue by JeanFred** [krobertson/deb-s3#73]
I’m trying to delete a package from my S3 APT repo.

`delete my_package -v authenticated -b my-apt-repo -c stage --arch amd64 --versions 0.90.1`
```
>> Versions to delete: 0.90.1
>> Retrieving existing manifests
/var/lib/gems/1.9.1/gems/deb-s3-0.7.0/lib/deb/s3/manifest.rb:72:in `+': can't convert nil into String (TypeError)
	from /var/lib/gems/1.9.1/gems/deb-s3-0.7.0/lib/deb/s3/manifest.rb:72:in `block in delete_package'
	from /var/lib/gems/1.9.1/gems/deb-s3-0.7.0/lib/deb/s3/manifest.rb:67:in `select'
	from /var/lib/gems/1.9.1/gems/deb-s3-0.7.0/lib/deb/s3/manifest.rb:67:in `delete_package'
	from /var/lib/gems/1.9.1/gems/deb-s3-0.7.0/lib/deb/s3/cli.rb:390:in `delete'
	from /var/lib/gems/1.9.1/gems/thor-0.18.1/lib/thor/command.rb:27:in `run'
	from /var/lib/gems/1.9.1/gems/thor-0.18.1/lib/thor/invocation.rb:120:in `invoke_command'
	from /var/lib/gems/1.9.1/gems/thor-0.18.1/lib/thor.rb:363:in `dispatch'
	from /var/lib/gems/1.9.1/gems/thor-0.18.1/lib/thor/base.rb:439:in `start'
	from /var/lib/gems/1.9.1/gems/deb-s3-0.7.0/bin/deb-s3:8:in `<top (required)>'
	from /usr/local/bin/deb-s3:19:in `load'
	from /usr/local/bin/deb-s3:19:in `<main>'
```

**Comment by krobertson** [comment]
Should be fixed by 5f5e545. I'll probably be releasing 0.7.1 here soon.

**Comment by JeanFred** [comment]
Thanks!
